### ISSUE TITLE
False positives in rspec change blocks

### ISSUE THREAD

**Issue by Goltergaul** [rubocop/rubocop-sequel#44]
rubocop-sequel thinks that rspec change expectations are database migrations:

e.g:

```
spec/app/repositories/run_session/strategy/mongo_db/hard_delete_spec.rb:67:41: C: Sequel/IrreversibleMigration: Avoid using document_count inside a change block. Use up & down blocks instead.
    it { expect { subject }.to change { document_count(user_id) }.by(-1) }

spec/app/repositories/sport_activity/wrappers/plausibility_spec.rb:225:60: C: Sequel/IrreversibleMigration: Avoid using find inside a change block. Use up & down blocks instead.
        expect { subject }.not_to(change { described_class.find(id: sport_activity.id).plausible })
```

**Comment by Goltergaul** [comment]
https://github.com/rubocop/rubocop-sequel/blob/master/lib/rubocop/cop/sequel/irreversible_migration.rb#L33 this should probably check further that the block is actually part of a squel migration

**Comment by cyberdelia** [comment]
You may also configure rubocop to only run sequel cops on migrations related code, it will also speed up your checks overall.

**Comment by Goltergaul** [comment]
I know but I don't really want to need to add custom config for such things. Should ideally just work out of the box

**Comment by AlasdairWallaceMackie** [comment]
I'll work on a fix for this, sorry about that!

**Comment by Goltergaul** [comment]
no problem at all, just wanted to bring it to your attention :)

**Comment by cyberdelia** [comment]
I believe this was fixed in 0.3.6.

**Comment by Goltergaul** [comment]
Thank you @cyberdelia
